mpas_analysis/ocean/nino34_index.py

```diff
@@ -154,7 +153,11 @@ def compute_nino34_index(regionSST, calendar):  # {{{
 
     anomaly = regionSST.groupby('month') - monthlyClimatology
 
-    return _running_mean(anomaly.to_pandas())  # }}}
+    # Remove the long term trend from the anomalies
+    detrendedAnomal = signal.detrend(anomaly.values)
```

Comment:
Can you try an xarray apply here?  This would be something like
`anomaly.apply(signal.detrend)`.  

This should make the code clearer and would require fewer lines of code being changed.